### ISSUE TITLE
Screen reader(Narrator) is reading incorrect count of columns in detailList

### ISSUE THREAD

**Issue by Harkesh030679** [microsoft/fluentui#12569]
Screen reader(Narrator) is reading incorrect count of columns in detailList

### Environment Information
OS: Win 10
Browser:  Edge

### Describe the issue:

Screen reader(Narrator) is reading incorrect count of columns in detailList, Please see below attached snap, You can see this snap contains 4 columns but narrator is reading as 6 columns(including 2 other columns i.e. radio and image)
So aria-colindex would be 4 rather than 6.
![DetailList](https://user-images.githubusercontent.com/62233836/78632610-184be300-78bd-11ea-9cf5-bebfad5d61f9.PNG)

### Please provide a reproduction of the issue in a codepen:
You can see same behavior on below link.
https://developer.microsoft.com/en-us/fluentui#/controls/web/detailslist/customitemcolumns

#### Actual behavior:

Screen reader(Narrator) is reading incorrect count of columns in detailList, Please see below attached snap, You can see this snap contains 4 columns but narrator is reading as 6 columns(including 2 other columns i.e. radio and image)
![DetailList](https://user-images.githubusercontent.com/62233836/78632610-184be300-78bd-11ea-9cf5-bebfad5d61f9.PNG)
#### Expected behavior:

So aria-colindex would be 4 rather than 6. so that narrator should read column count as 4 rather than 6, It means aria-colindex should be user defined.

**Comment by joschect** [comment]
@Harkesh030679 I think things might be working as expected. My understanding is that if there is visual information present the screen then the screen reader should be announcing that. Even though those two columns don't have titles, they are still present and can impact interaction. 

Is there a specific aria/accessibility guidance that you're going off of? Based on https://www.w3.org/TR/wai-aria-1.1/#aria-colindex it seems that the number of columns would normally be read out if we were using a standard table structure without col-index, which might  include the first two columns.

**Comment by Harkesh030679** [comment]
OK. But the columns which do not have title should be handled by developer or developer can set aria label of these if required

**Comment by joschect** [comment]
@Harkesh030679 The selection column does have an aria label. The thumbnail is missing one. I'm adding it in a pr.
